### DISCUSSION TITLE
Stuck on Serenity tutorial

### DISCUSSION THREAD

**Discussion by AaronRiboy** [serenity-is/Serenity#6610] [Help]
Hello, please help me if you know the way to solve. I have been doing this tutorial all the way from start but i got stuck here https://serenity.is/docs/tutorials/movies/allowing_multiple_genre_selection

 **From tutorial**
<img width="587" alt="image" src="https://user-images.githubusercontent.com/29535650/206355736-d8803b19-a716-44df-82c6-d02431a16bc8.png">

**My outcome error**
<img width="578" alt="image" src="https://user-images.githubusercontent.com/29535650/206356023-18e229b6-e24e-4419-b9ff-0dd4978069da.png">

Does anyone know why there is error on import .Thank you @volkanceylan

**Comment by VictorTomaili** [comment]
https://serenity.is/blog/2022/10/15/switching-to-es-modules
